contracts/rust/raw-incrementer/Cargo.toml

```diff
@@ -15,3 +15,4 @@ wee_alloc = "0.4"
 [profile.release]
 panic = "abort"
 lto = true
+opt= "z"
```

Comment:
Do we actually want to include this?

Reply:
No, that's a leftover of when Robin and I were comparing wasm sizes between ink! and the raw contracts -  removed.